assets/sass/imports/_header.scss

```diff
@@ -251,3 +251,85 @@
     top: $container-padding;
   }
 }
+
+
+.menu-header {
+    display: flex;
+    flex-direction: row;
+}
+
+.menu-header .menu-item-header {
+    display: flex;
+    align-items: center;
+    font-size: 16px;
+    height: 45px;
+    background-color: inherit;
+    width: max-content;
+}
+
+.menu-header .dropdown  {
+    display: block;
+    font-size: 16px;
+    padding: 11px;
+    position: absolute;
+    top: 60px;
+    min-width: 160px;
+}
+
+.menu-header .dropdown .dropdown-item {
+    display: none;
+    position: relative;
+    padding: 5px 10px;
+    right: 10px;
+    vertical-align: middle;
+    align-items: center;
+    box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.3);
+}
+
+.menu-header .dropdown-item {
+    display: none;
+    min-width: max-content;
+    height: 36px;
+    background-color: #fff;
+    margin: 0 auto;
+}
+
+.menu-header .dropdown-item:hover {
+    background-color: #f1f1f1;
```

Comment:
```suggestion
    background-color: #f6f6f6;
```
Lighten gray a bit

Reply:
I think it's harder to see this way. I have trouble.